### PR TITLE
NavBar Updates

### PR DESCRIPTION
**Pull request by cindyzhang12**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
- updated the name in the NavBar to the settings gear icon
- current NavBar text changes to grey if its selected

### How to Review
- look over NavBar.js & NavBar.css for both updates

### Related PRs
N/A

### Next steps
N/A

### Tests Performed, Edge Cases
N/A

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -29,30 +29,46 @@ class NavBar extends React.PureComponent {
...
+                    pathname === '/admin' ? 'navItemSelected' : 'navItem'
+                  }
+                >
                   <Link to="/admin">Admin</Link>
                 </li>
               )}
               <li className="navItem">
                 <Link to="/profile">
-                  <span>{displayName}</span>
+                  <img src={Gear} alt="Profile" />
```
Comment (ashleynguyen27) [inline src/components/NavBar.js]: Maybe alt can be changed to "Settings"
